packages/core/lib/command-utils.js

```diff
@@ -201,6 +203,41 @@ const runCommand = async function (command, options) {
   return await command.run(options);
 };
 
+const processHelpInput = inputStrings => {
+  //is it wrong syntax of help?
+  if (inputStrings.length > 2 && ["help", "--help"].includes(inputStrings[1])) {
+    const help = require("./commands/help").meta;
+    console.log(
+      "Error: please use below syntax to displaying help information\n",
```

Comment:
you probably want this to read something like "please use the following syntax for accessing help information" or "please use the syntax below in order to access help information"

Reply:
thank you. I will use one of these.